src/index.html

```diff
@@ -5,9 +5,39 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div data-qa="card" class="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="mac-picture"
+        class="card__image"
+      >
```

Comment:
```suggestion
      <img
        src="./images/imac.jpeg"
        alt="APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)"
        class="card__image"
      >
```
Checklist:
13. [SEO] - alt atribute should describe the image if the image contains information (better description you have - better for you :))